docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -0,0 +1,167 @@
+
+## Deleting and restoring dashboards
+
+To delete a dashboard:
+
+1. Go to the model you want to delete;
+2. Click on the three dots menu;
+3. Select "Move to trash".
+
+You'll still be able to see the contents of the dashboard in **Trash**, but you won't be able to modify it.
+
+To restore a dashboard:
+
+1. Go to Trash;
+2. Find the dashboard you'd like to delete. You can sort deleted items to make it easier to find the dashboard, or [search for your dashboard in Trash](#search-in-trash);
+3. Click on the checkbox next to the dashboard to select it;
+4. Select "Restore".
+
+The dashboard will be restored to the parent collection that it was most recently saved in. If the dashboards's original collection has been deleted as well, you won't see an option to **Restore**, but you'll still be able to move the dashboard from Trash to a different collection.
+
+Deleting or restoring a dashboard will affect other items that depend on that dashboard:
+
+- **Dashboards subscriptions**: If a dashboard is deleted, any subscriptions to the dashboard will be deactivated. If a dashboard is restored from Trash, the subscriptions will be restored as well.
+
+- [**Custom home page**](../configuring-metabase/appearance.md#landing-page): If a deleted dashboard was set as a custom home page, the home page will revert to the default Metabase home page. If a dashboard is restored, the custom home page will be restored as well.
+
+## Deleting and restoring models
+
+To delete a model:
+
+1. Go to the model you want to delete;
+2. Click on the three dots menu;
+3. Select "Move to trash".
+
+You'll still be able to see the model in Trash, but you won't be able to modify it.
+
+To restore a model:
+
+1. Go to Trash;
+2. Find the model you'd like to delete;
+3. Click on the three dot menu;
+4. Select "Restore".
+
+Deleting a model will affect other items that depend on that model:
+
+- **Derived questions**: If a trashed model was used as data source for a Query Builder question, or was [referenced in a SQL question](../questions/native-editor/referencing-saved-questions-in-queries.md), then the derived questions will continue to work while the model is in Trash. If the model is [permanently deleted](#permanently-deleting-items), the derived questions will stop working, and you'll get a `Card does not exist` error.
+
+- **Actions**: If a model is trashed, the actions in this model will still work. If a model is permanently deleted, the actions in the deleted model will be deleted as well.
+
+- **Dashboards**: If a model is included as a card in any dashboards, then the card will be removed from those dashboards when the model is moved to Trash. If the model is restored from Trash, its card will be restored to the dashboards that used to contain it.
```

Comment:
The text is confusing. maybe a table? Columns are: Model in collection, Trash, permanently deleted.

So something like: 

**Item** |**In trash**|**Permanently deleted**|**Restored**
-----|-----|-----|-----
Question based on model|Works as normal|Breaks|Works as normal
Dashboard that includes model|Model card is removed from the dashboard| Model card is removed from the dashboard |Card is restored to the dashboard